package.json

```diff
@@ -26,43 +26,44 @@
     "url": "https://github.com/rangle/angular-2-aot-sandbox/issues"
   },
   "dependencies": {
```

Comment:
That peerDependency warning can be fixed if you change the line `@ngtools/webpack: ^1.1.5` to `@ngtools/webpack: ^1.1.7`.